### PR TITLE
add margins between elements inside a list item

### PR DESCRIPTION
**Pull request by gbicou**
a style rule to add spacing between children of li elements (for example when there is multiple paragraphs inside a list item)

### CODE REVIEW

File: src/interface/interface.vue [hunk 1]
```diff
@@ -307,6 +307,7 @@
     }
 
     /* Editor Styles */
+    .flexible-editor :deep(.ProseMirror li > * ~ *),
```
Comment (formfcw) [inline src/interface/interface.vue:310]: I think we should make this more specific and add more complex selectors, to target some edge cases:

```css
.flexible-editor .ProseMirror li > * ~ :not(ul):not(ol),
.flexible-editor .ProseMirror li > * ~ :not(ul):not(ol) ~ *,
```

The above code will only add the margin to sibling children that are not ul/ol elements, unless one already has a margin.

Before:
before

After:
after

I think it looks cleaner.

**Review by formfcw — COMMENTED** [review]
Thanks for contributing!

File: src/interface/interface.vue (hunk 1 above)
Comment (formfcw) [reply src/interface/interface.vue:310]: @gbicou Want to update your PR?

Comment (gbicou) [reply src/interface/interface.vue]: updated

**Review by formfcw — APPROVED** [review]

**Pull request merged**